Cargo.toml

```diff
@@ -281,4 +281,4 @@ debug = true
 # egui-wgpu = { path = "../../egui/crates/egui-wgpu" }
 # emath = { path = "../../egui/crates/emath" }
 
-egui_tiles = { git = "https://github.com/rerun-io/egui_tiles", rev = "dcdcde10b493683f3e97db47d128683bec0e65b9" } # https://github.com/rerun-io/egui_tiles/pull/40
+egui_tiles = { git = "https://github.com/rerun-io/egui_tiles", rev = "54dac21814f53df1b2fbd152643b93273ebbb89a" } # https://github.com/rerun-io/egui_tiles/pull/41
```

Comment:
Using commit hashes to feature branches is dangerous! When a branch is squashed-merged the original branch may be deleted, and the commit can be get lost. This means we may end up with a version of Rerun that doesn't compile, which is horrible when using `git bisect`, among other things.

In this case at least it is our own repository, so we have a bit more control over fixing it, but generally it is better if we merge the PR before depending on it.

Reply:
That makes sense.